nemo/collections/asr/modules/conv_asr.py

```diff
@@ -468,6 +468,82 @@ def num_classes_with_blank(self):
         return self._num_classes
 
 
+class ConvASRDecoderRecon(NeuralModule, Exportable):
+    """ASR Decoder for reconstructing masked regions of spectrogram
+    """
+
+    def save_to(self, save_path: str):
+        pass
+
+    @classmethod
+    def restore_from(cls, restore_path: str):
+        pass
+
+    @property
+    def input_types(self):
+        return OrderedDict({"encoder_output": NeuralType(('B', 'D', 'T'), AcousticEncodedRepresentation())})
+
+    @property
+    def output_types(self):
+        return OrderedDict({"spec_recon": NeuralType(('B', 'T', 'D'), SpectrogramType())})
+
+    def __init__(self, feat_in, feat_out, feat_hidden, stride_layers, kernel_size=11, init_mode="xavier_uniform"):
+        super().__init__()
+
+        self.feat_in = feat_in
+        self.feat_out = feat_out
+        self.feat_hidden = feat_hidden
+
+        self.decoder_layers = [nn.Conv1d(self.feat_in, self.feat_hidden, kernel_size=1, bias=True)]
+        for i in range(stride_layers):
+            self.decoder_layers.append(nn.ReLU())
```

Comment:
No choice for activation? Swish is continuously differentiable and would be good to experiment with.